templates/repo/diff/compare.tmpl

```diff
@@ -1,8 +1,8 @@
 {{template "base/head" .}}
 <div role="main" aria-label="{{.Title}}" class="page-content repository diff {{if .PageIsComparePull}}compare pull{{end}}">
 	{{template "repo/header" .}}
-	<div class="ui container fluid padded">
```

Comment:
Just found that for this page, the widths of containers are different now, maybe we should still keep the `fluid padded` classes here?

<img width="1433" alt="Screen Shot 2023-05-10 at 10 19 03" src="https://github.com/go-gitea/gitea/assets/17645053/4396cae8-8e25-4ecc-9dbb-4b48ffa84ed5">

Reply:
That's done by purpose: "And this PR also make the "create new PR form" not that wide"

Otherwise the "form" will be too wide.

Reply:
Github does not use wider width for these pages(diff and new pr), maybe we can do the same? remove the fluid on these two pages.

dff page:
<img width="1429" alt="Screen Shot 2023-05-10 at 10 35 39" src="https://github.com/go-gitea/gitea/assets/17645053/3290a737-26ed-4fcc-8f3d-d263573f30cd">

new pr page:
<img width="1428" alt="Screen Shot 2023-05-10 at 10 37 04" src="https://github.com/go-gitea/gitea/assets/17645053/8b6cf9bb-d2da-4cc5-b99c-0d8a005c28a6">
<img width="1424" alt="Screen Shot 2023-05-10 at 10 35 53" src="https://github.com/go-gitea/gitea/assets/17645053/aa64df91-6dc2-4ef1-ae72-4b3afccaaa1f">

Reply:
Good point, then we can make this page narrow.